test/queue-test.js

```diff
@@ -877,6 +877,88 @@ describe('Queue', (it) => {
     });
   });
 
+  it.describe('saveAll', (it) => {
```

Comment:
Mind adding a test that shows when `saveAll` rejects?

Reply:
Done!